packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -346,8 +346,8 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
         transition: `opacity ${AnimationVariables.durationValue3} ${AnimationVariables.easeFunction2}`
       },
       isFooterSticky && {
-        background: palette.white,
-        borderTopColor: palette.neutralLight
+        background: semanticColors.bodyBackground,
```

Comment:
does it make sense to user a body slot for the footer here? (just asking out of curiosity - it really might make sense to do this but wondering if that's the pattern we're meant to follow in other places throughout the codebase too)

Reply:
Hi @aneeshack4 do you have more context on how the pattern different? And what is being used everywhere else?

Reply:
I don't have the full context but based on the recent PRs I've seen using semantic slots, it seems like the usage varies a lot. I think body makes the most sense here actually even though words wise it's a bit weird to use body on footer but there are no better options from the semantic slots list.

Reply:
Yeah I see what you mean, wording-wise `body` is weird to use on a `footer`..seems to be a special case since it's a sticky footer we need that white BG layer to overlay on top of panel content when scrolling

Reply:
bodyBackground works for me. the sticky footer needs a bg that matches the rest of the panel so that it can sit on top of panel content scrolling under